.buildkite/pipeline.yml

```diff
@@ -23,6 +23,7 @@ steps:
     command: |
       source ~/.cargo/env && set -eux
       rustc --version && cargo --version
+      RUSTFLAGS='-D warnings' cargo run -p themis
```

Comment:
RUST_LOG=WARN

Reply:
`RUSTFLAGS='-D warnings'` is for the compiler. `RUST_LOG=WARN` is for runtime related errors, like with `log`, `eyre`, `tracing`, etc..

Reply:
To be clear, for example: while `cargo run` typically warns when you have unused variables, even though it would build and run just fine, when you add `RUSTFLAGS="-D warnings"` then it would fail and cause a hard error.

Reply:
@miraclx Oh, sorry. You are right.